src/data/constants.js

```diff
@@ -50,3 +50,5 @@ export const RESERVED_QUICKSTART_IDS = {
   GUIDED_INSTALL: 'GUIDED_INSTALL',
   BUILD_YOUR_OWN_QUICKSTART: 'BUILD_YOUR_OWN_QUICKSTART',
 };
+
+export const COLLAPSE_BREAKPOINT = '760px';
```

Comment:
Okay, I'm so sorry, but let's call this something with quickstarts in the name since that's the only place it's used